controllers/entityController.go

```diff
@@ -69,6 +69,15 @@ func DispRequestMetaData(r *http.Request) {
 	fmt.Println(time.Now().Format("2006-Jan-02 Monday 03:04:05 PM MST -07:00"))
 }
 
+func getFiltersFromQueryParams(r *http.Request) map[string][]string {
+	filters := map[string][]string{}
+	r.ParseForm()
+	filters["fields"] = r.Form["fieldOnly"]
+	filters["startDate"] = r.Form["startDate"]
+	filters["endDate"] = r.Form["endDate"]
+	return filters
+}
```

Comment:
We could have a struct for the filters rather than a map.

Reply:
Good idea, will be useful for future implementations of new filters, I'll commit it soon.